lib/iris/fileformats/rules.py

```diff
@@ -1014,15 +1019,6 @@ def load_cubes(filenames, user_callback, loader, filter_function=None):
     if isinstance(filenames, six.string_types):
         filenames = [filenames]
 
-    def _generate_all_fields_and_filenames():
-        for filename in filenames:
-            for field in loader.field_generator(
-                filename, **loader.field_generator_kwargs):
-                # evaluate field against format specific desired attributes
-                # load if no format specific desired attributes are violated
-                if filter_function is None or filter_function(field):
-                    yield (field, filename)
-
```

Comment:
@dkillick This private functions doesn't appear to have changed, apart from its position in the file.

Sorry for being a pain, but could you paste it back, as it introduces unnecessary noise, thanks.